libs/templates/src/lib/trade/chips/index.tsx

```diff
@@ -14,37 +15,34 @@ const Chips = ({ onChangeChips, onChangeSelectedInfo }: ChipsProps) => {
   return (
     <>
       <Chip.Selectable
-        className="hidden md:block lg:block"
         onChipSelect={() => {
           onChangeSelectedInfo('mainInfo');
           setSelectedInfo('mainInfo');
           onChangeChips(mainInfo.data);
         }}
         selected={selectedInfo === 'mainInfo'}
       >
-        Main Info
+        <div className="w-max">Main Info</div>
       </Chip.Selectable>
       <Chip.Selectable
-        className="hidden md:block lg:block"
         onChipSelect={() => {
           onChangeChips(tradingCondition.data);
           onChangeSelectedInfo('tradingCondition');
           setSelectedInfo('tradingCondition');
         }}
         selected={selectedInfo === 'tradingCondition'}
       >
-        Trading conditions
+        <div className="w-max">Trading conditions</div>
       </Chip.Selectable>
       <Chip.Selectable
-        className="hidden md:block lg:block "
         onChipSelect={() => {
           onChangeChips(additionalInfo.data);
           onChangeSelectedInfo('additionalInfo');
           setSelectedInfo('additionalInfo');
         }}
         selected={selectedInfo === 'additionalInfo'}
       >
-        Additional info
+        <div className="w-max">Additional info</div>
```

Comment:
Why do we need this?

Reply:
on some responsive views the content inside chips is taking two lines. This should be fixed from quill-design.